spec/batch_loader_spec.rb

```diff
@@ -299,7 +299,7 @@
         items.map! { |i| i - 1 }
       end
 
-      expect { batch_loader.to_s }.to raise_error(RuntimeError, "can't modify frozen Array")
+      expect { batch_loader.to_s }.to raise_error(RuntimeError, /\Acan't modify frozen Array/)
```

Comment:
this needs to be changed due to: https://bugs.ruby-lang.org/issues/15883

(Just in case I described the reason behind this on the commit log as well)